src/main.js

```diff
@@ -119,12 +66,64 @@ function generateUserCover() {
   descriptors.push(userDescriptor1, userDescriptor2)
 }
 
+function goHome() {
+  homeView.classList.remove('hidden');
+  formView.classList.add('hidden');
+  savedView.classList.add('hidden');
+  saveCoverBtn.classList.remove('hidden');
+  homeBtn.classList.add('hidden');
+  newRandomCoverBtn.classList.remove('hidden');
+}
+
+function goToMakeCoverInputPage() {
+  savedView.classList.add('hidden')
+  homeView.classList.add('hidden');
+  homeBtn.classList.remove('hidden');
+  newRandomCoverBtn.classList.add('hidden');
+  saveCoverBtn.classList.add('hidden');
+  formView.classList.remove('hidden');
+}
+
 function saveCover() {
   if (!savedCovers.includes(currentCover)) {
     savedCovers.push(currentCover);
   }
 }
 
+function seeSavedCovers() {
+  savedView.classList.remove('hidden');
+  formView.classList.add('hidden');
+  homeView.classList.add('hidden');
+  newRandomCoverBtn.classList.add('hidden');
+  saveCoverBtn.classList.add('hidden');
+  homeBtn.classList.remove('hidden');
+  savedCoversSection.innerHTML = "";
+  for (var i = 0; i < savedCovers.length; i++) {
+      savedCoversSection.innerHTML += `
+      <section class="main-cover mini-cover" id="${savedCovers[i].id}">
+        <img class="cover-image" src=${savedCovers[i].cover}>
+        <h2 class="cover-title">${savedCovers[i].title}</h2>
+        <h3 class="tagline">A tale of <span class="tagline-1">${savedCovers[i].tagline1}</span> and <span class="tagline-2">${savedCovers[i].tagline2}</span></h3>
+        <img class="price-tag" src="./assets/price.png">
+        <img class="overlay" src="./assets/overlay.png">
+      </section>
+      `
+    }
+}
+
+function showNewCover() {
+  var newImage = covers[getRandomIndex(covers)];
+  var newTitle = titles[getRandomIndex(titles)];
+  var newDescriptor1 = descriptors[getRandomIndex(descriptors)];
+  var newDescriptor2 = descriptors[getRandomIndex(descriptors)];
+
+  currentCover = new Cover (newImage, newTitle, newDescriptor1, newDescriptor2);
+
+  displayCover.src = newImage;
```

Comment:
I would use currentCover.src here instead of using the newImage. It doesn't matter much but just to keep the currentCover in more use. You could also use a helper function an pass in the currentCover to fill the display cover.

Reply:
Awesome! Thanks, Taylor! You're right that a couple of these are getting a bit long and this could be a good way to clean them up. I'm going to look at doing that right now.